src/index.html

```diff
@@ -11,7 +11,197 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form class="lines"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      name="form"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div  class="form-field">
+          <label for="Surename">Surname:
+
+            <input type="text"
+              id="Surename"
+              name="Surname"
+              autocomplete="off"
+            >
```

Comment:
```suggestion
          <label for="Surename">Surname:
            <input
              type="text"
              id="Surename"
              name="Surname"
              autocomplete="off"
            >
```

No need to add blank line between parend and child elements

Fix all such cases